pkgs/top-level/all-packages.nix

```diff
@@ -15013,6 +15013,11 @@ with pkgs;
     backend = "wayland";
   };
 
+  rofi-network-manager = callPackage ../by-name/ro/rofi-network-manager/package.nix { };
+  rofi-network-manager-wayland = callPackage ../by-name/ro/rofi-network-manager/package.nix {
+    backend = "wayland";
+  };
+
```

Comment:
For all packages in the `pkgs/by-name/` tree, there is no longer a need to explicitly declare packages here.

```suggestion
```